sklearnex/basic_statistics/tests/test_basic_statistics.py

```diff
@@ -0,0 +1,251 @@
+# ==============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+
+from onedal.basic_statistics.tests.test_basic_statistics import (
+    expected_max,
+    expected_mean,
+    expected_sum,
+    options_and_tests,
+)
+from onedal.tests.utils._dataframes_support import (
+    _convert_to_dataframe,
+    get_dataframes_and_queues,
+)
+from sklearnex.basic_statistics import BasicStatistics
+
+
+@pytest.mark.parametrize("dataframe,queue", get_dataframes_and_queues())
+def test_sklearnex_import_basic_statistics(dataframe, queue):
+    X = np.array([[0, 0], [1, 1]])
+    X_df = _convert_to_dataframe(X, sycl_queue=queue, target_df=dataframe)
+
+    weights = np.array([1, 0.5])
+    weights_df = _convert_to_dataframe(weights, sycl_queue=queue, target_df=dataframe)
+
+    result = BasicStatistics().fit(X_df)
+
+    expected_mean = np.array([0.5, 0.5])
+    expected_min = np.array([0, 0])
+    expected_max = np.array([1, 1])
+
+    assert_allclose(expected_mean, result.mean)
+    assert_allclose(expected_max, result.max)
+    assert_allclose(expected_min, result.min)
+
+    result = BasicStatistics().fit(X_df, sample_weight=weights_df)
+
+    expected_weighted_mean = np.array([0.25, 0.25])
+    expected_weighted_min = np.array([0, 0])
+    expected_weighted_max = np.array([0.5, 0.5])
```

Comment:
Am I right, even dpctl/dpnp given, results are numpy arrays?

Reply:
I expect that we have full interobility.
If dpnp/dpctl given, the same dataframe type with the same sycl context are expected as results

Reply:
I see that this common question for some other estimators as well. Make sense to work on this separately. Covered by ticket.